backend/app/services/adapters/task_kinds.py

```
@@ -1169,23 +1193,27 @@ def get_user_tasks_by_title_with_pagination(
         # Use raw SQL to get task IDs without JSON_EXTRACT in WHERE clause
+        # Exclude system namespace tasks (background tasks)
         count_sql = text(
             """
             SELECT COUNT(*) FROM tasks
             WHERE user_id = :user_id
             AND kind = 'Task'
             AND is_active = true
+            AND namespace != 'system'
         """
         )
         total_result = db.execute(count_sql, {"user_id": user_id}).scalar()
```

Ruff (0.14.11)

Local variable `total_result` is assigned to but never used

Remove assignment to unused variable `total_result`

(F841)